xtask/codegen/js.ungram

```diff
@@ -1237,7 +1237,7 @@ AnyJsImportClause =
 // import "abcd" assert ...
 JsImportBareClause =
 	source: AnyJsModuleSource
-	assertion: JsImportAssertion?
+	attribute: JsImportAttribute?
```

Comment:
I wonder if we should rename `JsImportAttribute` into `JsImportWithClause` and `attribute into `with_clause`:

```suggestion
	with_clause: JsImportWithClause?
```

What do you think?